tests/DoctrineTest.php

```diff
@@ -258,23 +274,24 @@ public static function autoload($class)
         }
 
         $e = explode('_', $class);
+
         $count = count($e);
         $prefix = array_shift($e);
 
         if ($prefix !== 'Doctrine') {
             return false;
         }
 
-        $dir = array_shift($e);
+        $dir = DOCTRINE_DIR . '/tests/' . array_shift($e);
```

Comment:
I did this because if you run the tests from the application root, it dumps a bunch of files in the root, this prevents that from happening (but some tests will still fail if the test runner isn't initiated from the `tests` dir).

I believe those tests could be fixed to make this possible, but not really worth the time.